app/src/main/java/nl/tudelft/pixelperfect/client/EventsMessage.java

```diff
@@ -0,0 +1,38 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.AbstractMessage;
+import com.jme3.network.serializing.Serializable;
+
+/**
+ * The class that is sent by the server when it shares an event.
+ *
+ * @author Dmitry Malarev.
+ */
+@Serializable
+public class EventsMessage extends AbstractMessage{
+    private String event;
```

Comment:
Because, as things are right now, an event is difficult to serialize without reworking the Event class